Neos.Utility.Unicode/Classes/TYPO3/Flow/Utility/Unicode/Functions.php

```diff
@@ -11,29 +11,28 @@
  * source code.
  */
 
-use TYPO3\Flow\Annotations as Flow;
 
 /**
  * A class with UTF-8 string functions, some inspired by what might be in some
  * future PHP version...
  *
- * @Flow\Scope("singleton")
  * @api
  */
-class Functions
+abstract class Functions
```

Comment:
It's something I personally prefer for classes that contain only static methods. Just to prevent instanciation. Can be removed of course.